idaes_examples/notebooks/docs/flowsheets/temperature_swing_adsorption/temperature_swing_adsorption.ipynb

```diff
@@ -0,0 +1,645 @@
+    "\n",
+    "1. `dynamic`: to set up the model as steady state. The IDAES fixed bed TSA 0D\n",
+    "   unit model only supports steady state as the dynamic nature of the adsorption\n",
+    "   cycle is handled in internal blocks for each cycle step of the unit. This\n",
+    "   config argument is used to enable the TSA unit model to connect with other\n",
+    "   IDAES unit models.\n",
+    "2. `adsorbent`: to set up the adsorbent to be used in the fixed bed TSA system. \n",
+    "   Supported values currently are `Adsorbent.zeolite_13x`, `Adsorbent.mmen_mg_mof_74`, and `Adsorbent.polystyrene_amine`.\n",
+    "3. `number_of_beds`: to set up the number of beds to be used in the unit model.\n",
+    "   This config argument accepts either an `int` (model assumes a fixed number     of beds) or `None` (model calculates the number of beds).\n",
+    "4. `compressor`: indicates whether a compressor unit should be added to the\n",
+    "   fixed bed TSA system to calculate the energy required to overcome\n",
+    "   the pressure drop in the system. Supported values are `True` and `False`.\n",
+    "5. `compressor_properties`: indicates a property package to use in the compressor unit model.\n",
+    "6. `steam_calculation`: indicates whether a method to estimate the steam flow rate\n",
+    "   required in the desorption step should be included. Supported values are: `SteamCalculationType.none`,\n",
+    "   steam calculation method is not included. `SteamCalculationType.simplified`, a surrogate model is used\n",
+    "   to estimate the mass flow rate of steam. `SteamCalculationType.rigorous`, a heater unit model is\n",
+    "   included in the TSA system assuming total saturation.\n",
+    "7. `steam_properties`: indicates a property package to use for rigorous steam calculations. Currently, only the iapws95 property package is supported.\n",
+    "8. `transformation_method`: to set up the discretization method to be use for the time\n",
+    "   domain. The discretization method must be a method recognized by the\n",
+    "   Pyomo `TransformationFactory`. Supported values are `dae.finite_difference` and\n",
+    "   `dae.collocation`.\n",
+    "9. `transformation_scheme`: to set up the scheme to use when discretizing the time domain.\n",
+    "   Supported values are: `TransformationScheme.backward` and `TransformationScheme.forward` for finite difference transformation\n",
+    "   method. `TransformationScheme.lagrangeRadau` for collocation transformation method.\n",
+    "10. `finite_elements`: to set up the number of finite elements to use when discretizing\n",
+    "   the time domain.\n",
+    "11. `collocation_points`: to set up the number of collocation points to use per finite element\n",
+    "    when the discretization method is `dae.collocation`.\n",
+    "  \n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "<b>Note:</b>  a default value defined in every IDAES unit class is used for\n",
```

Comment:
Minor grammar: I would use "the" instead of "every".

Reply:
changed